lang/python/python-yaml/Makefile

```diff
@@ -8,36 +8,62 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=PyYAML
-PKG_VERSION:=3.11
+PKG_VERSION:=3.12
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://pyyaml.org/download/pyyaml/
-PKG_MD5SUM:=f50e08ef0fe55178479d3a618efe21db
+PKG_MD5SUM:=4c129761b661d181ebf7ff4eb2d79950
+PKG_HASH:=592766c6303207a20efc445587778322d7f73b161bd994f227adaa341ba212ab
 
 PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=COPYING
 PKG_MAINTAINER:=Nicolas Thill <nico@openwrt.org>
 
 PKG_BUILD_DEPENDS:=python
 
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-$(PKG_NAME)-$(PKG_VERSION)
+PKG_UNPACK=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
 include $(INCLUDE_DIR)/package.mk
 $(call include_mk, python-package.mk)
+$(call include_mk, python3-package.mk)
 
-define Package/python-yaml
+define Package/python-yaml/Default
   SUBMENU:=Python
   SECTION:=lang
   CATEGORY:=Languages
-  TITLE:=YAML parser and emitter for Python
   URL:=http://pyyaml.org/wiki/PyYAML
+endef
+
+define Package/python-yaml
+$(call Package/python-yaml/Default)
+  TITLE:=YAML parser and emitter for Python
   DEPENDS:=+python +libyaml
```

Comment:
you could move the `DEPENDS:=+libyaml` to the  `Package/python-yaml/Default` rule
and do  `DEPENDS+=+PACKAGE_python-yaml:python` here and `DEPENDS+=+PACKAGE_python3-yaml:python3`  to the Python3 variant

also, I encourage the use of the `+PACKAGE_python-yaml:python` & `+PACKAGE_python3-yaml:python3` style variables ;
i noticed, that if you just specify `+python` & `+python3` dependencies, both Python & Python3 interpreters get built, even tho, one gets installed on the target [ or packaged ]
when both interpreters get built, that also means that the host Python & Python3 interpreters get built, so the build time reduction can be significant

this is a general thing with the LEDE/OpenWrt build system, and maybe could be addressed in core ;
but, until then, to speed up builds, this is one way to do it